tests/test_pandas_api.py

```diff
@@ -1489,6 +1489,89 @@ def test_df_sample(kx, q):
         t.sample(ignore_index=True)
 
 
+def test_sem(kx, q):
+    df = pd.DataFrame(
+        {
+            'a': [1, 2, 2, 4],
+            'b': [1, 2, 6, 7],
+            'c': [7, 8, 9, 10],
+            'd': [7, 11, 14, 14]
+        }
+    )
+    tab = kx.toq(df)
+    p_m = df.sem()
+    q_m = tab.sem()
+    assert all([p_m[c] == pytest.approx(q_m[c].py(),  1e-16)

```

Comment:
Minor: The value `1e-16` keeps being used throughout the tests. Maybe we could keep it in a variable `precision` or something similar. What do you think.